any.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 Gin Core Team.  All rights reserved.
+// Use of this source code is governed by a MIT style
+// license that can be found in the LICENSE file.
+
+//go:build !go1.18
+// +build !go1.18
```

Comment:
May be invert?
//go:build go1.18
// +build go1.18

binding.go:33:22: undeclared name: any (requires version go1.18 or later)

Reply:
+1, 
this caused the above issue if the machine has go version 1.18 but the app that imports gin uses version lower than 1.18